src/platform/user/profile/vet360/containers/AddressValidationModal.jsx

```diff
@@ -0,0 +1,179 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import { get, map } from 'lodash';
```

Comment:
we can use native Array.map and ES6 optional chaining instead of lodash

Reply:
I removed the use of get, but would like to use map.. here is the code comparison.

```
{map(addresses, (address, index) =>
            this.renderAddressOption(address, String(index)),
          )}
```

would become: 

```
const content = this.props.address.map((address, index) => 

this.renderAddressOption(address, String(index)),

render(...stuff
  {content}
)
```

Reply:
@erikphansen what do you think?

Reply:
It also comes with added benefit of not having ANY potential of throwing runtime errors of 'Cannot find map of undefined'